src/java.base/share/classes/java/text/DecimalFormatSymbols.java

```diff
@@ -1148,6 +1147,11 @@ private void readObject(ObjectInputStream stream)
     private transient Currency currency;
     private transient volatile boolean currencyInitialized;
 
+    /**
+     * Cached hash code
+     */
+    private volatile int hashCode;
```

Comment:
I'm thinking the cached hashcode should be / should have been transient (not serialized).
And add a period at the end of the sentence above.

Reply:
Good point. Addressed both comments.